makefile

```diff
@@ -91,6 +109,8 @@ clean_docs:
 	rm -rf docs/tutorials
 	rm -rf docs/source/apiref
 	rm -rf docs/source/tutorials
+	rm -rf docs/source/_static/drawio
+	rm -rf docs/source/diagrams/**/export
```

Comment:
Could you make the patterns consistent? (`.gitignore` uses `docs/source/diagrams/*/export`, but here it has two stars)

Reply:
I don't think we need that because in the future (when the bash script will be rewritten in python) I would like it to support any nested structure, not just one layer.

Reply:
Could you still use either one start or two stars in all files? It would be easier if we used the same file path pattern for the same files.